tests/bcl-test/templates/common/TestRunner.Core/TcpTextWriter.cs

```diff
@@ -62,22 +64,40 @@ static string SelectHostName (string[] names, int port)
 			return result;
 		}
 
-		public TcpTextWriter (string hostName, int port)
+		public TcpTextWriter (string hostName, int port, bool isTunnel = false)
```

Comment:
FYI matches the default in Touch.Unit: https://github.com/spouliot/Touch.Unit/pull/56